Duplicati/Library/Snapshots/WindowsSnapshot.cs

```diff
@@ -108,15 +108,12 @@ public WindowsSnapshot(string[] sourcepaths, Dictionary<string, string> options)
                 }
 
                 //Sanity check for duplicate files/folders
-                var pathDuplicates = m_sourcepaths.GroupBy(x => x, Utility.Utility.ClientFilenameStringComparer)
-                      .Where(g => g.Count() > 1).Select(y => y.Key).ToList();
+                ISet<string> pathDuplicates;
+                m_sourcepaths = Utility.Utility.GetUniqueItems(m_sourcepaths, Utility.Utility.ClientFilenameStringComparer, out pathDuplicates).ToList();

```

Comment:
The original implementation sorted the results, but only if the collection contained duplicates.  I interpreted this to mean that the result does not really need to be sorted, so I removed the `OrderBy` clause.

Reply:
I think that is correct, there is no need for sorting. The paths are only used to figure out which disks to make VSS snapshots from.